x/mint/keeper/emissions.go

```diff
@@ -19,45 +24,75 @@ func GetLockedVestingTokens(
 	blockHeight math.Int,
```

Comment:
```suggestion
	blockCount math.Int,
```

This parameter is actually used as a block count since TGE so I'd rather rename it as this is a bit misleading

Reply:
yeah this pattern then should be applied to other functions too like GetCirculatingSupply